src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -73,22 +81,21 @@ public static void Record(string assemblyFileName, string sessionName, string ou
             WriteInfoLine($"ETW Tracing Session saved to \"{userFullFileName}\"");
             collectOutputFilesCallback(userFullFileName);
 
-            var assemblyModel = GetAssemblyModel(assemblyFileName, userFullFileName, sessionName, performanceTestMessages);
-            var fileNameWithoutExtension = Path.GetFileNameWithoutExtension(userFullFileName);
-            var xmlFileName = Path.Combine(outputDirectory, $"{fileNameWithoutExtension}.xml");
+            var assemblyModel = GetAssemblyModel(assemblyFileName, userFullFileName, runId, performanceTestMessages);
+            var xmlFileName = Path.Combine(outputDirectory, $"{name}.xml");
             new AssemblyModelCollection { assemblyModel }.Serialize(xmlFileName);
             WriteInfoLine($"Performance results saved to \"{xmlFileName}\"");
             collectOutputFilesCallback(xmlFileName);
 
-            var mdFileName = Path.Combine(outputDirectory, $"{fileNameWithoutExtension}.md");
+            var mdFileName = Path.Combine(outputDirectory, $"{name}.md");
```

Comment:
It looks like you have some other file name changes in this PR too.  Are these intentional?